client/src/ui/components/Modal.tsx

```diff
@@ -1,99 +1,71 @@
 import React from 'react';
-import { Button } from './';
+import { Button } from './Button';
 
 interface ModalProps {
-  closeModalButton: string;
-  ctaButton: string;
-  mainText: React.ReactNode;
-  header: string;
-  closeModal: () => void;
-  ctaFn: () => void;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+export const Modal = ({ children, className }: ModalProps) => (
+  <div className="bg-gray-400 bg-opacity-25 absolute inset-0 flex justify-center items-center">
+    <div className="bg-white rounded-md shadow-xl w-156 py-6 px-6">
+      {children}
+    </div>
+  </div>
+);
+
+interface ModalTitleProps {
+  children?: React.ReactNode;
+  className?: string;
+}
+
+export const ModalTitle = ({ children, className }: ModalTitleProps) => (
+  <div className="flex justify-between items-center">
+    <h3 className="text-lg font-medium">{children}</h3>
+  </div>
+);
+
+interface ModalDescriptionProps {
+  children?: React.ReactNode;
+  className?: string;
```

Comment:
@Akirtovskis className is still allowed but if you pass one currently it will not work. You will need to merge this prop with the current className

Reply:
@pradel should be good now after the latest commit.